pom.xml

```diff
@@ -304,15 +305,9 @@
                 </exclusion>
             </exclusions>
         </dependency>
-        <dependency>
-            <groupId>org.bouncycastle</groupId>
-            <artifactId>bcprov-jdk15on</artifactId>
-            <version>1.70</version>
-            <scope>test</scope>
-        </dependency>
         <dependency>
             <groupId>org.hamcrest</groupId>
-            <artifactId>hamcrest-core</artifactId>
```

Comment:
Unless you are 100% certain that hamcrest and hamcrest-core are identical and always will be, let's just leave this as-is too. hamcrest-core is legit (see https://mvnrepository.com/artifact/org.hamcrest/hamcrest-core/2.2). It just seemed like 'hamcrest-core' would be minimal core classes and 'hamcrest' would be that and extra stuff that we are not using.

Reply:
Restored